src/main/java/TaskManager.java

```diff
@@ -0,0 +1,59 @@
+public class TaskManager {
+    private Task[] taskList = new Task[100];
+    private int taskCount = 0;
+    protected String byString = " /by ";
+    protected String atString = " /at ";
+
+    public Task getTaskType(String newTask, TaskType type) {
+        switch (type) {
+        case ADD:
+            return new Task(newTask);
+        case TODO:
+            return new Todo(newTask);
+        case DEADLINE:
+            String[] taskDetailsArr = getTaskNameAndTime(newTask, byString);
+            return new Deadline(taskDetailsArr[0], taskDetailsArr[1]);
+        case EVENT:
+            String[] taskDetailsArr2 = getTaskNameAndTime(newTask, atString);
+            return new Event(taskDetailsArr2[0], taskDetailsArr2[1]);
+        }
+        return null;
+    }
+
+    public String[] getTaskNameAndTime(String taskDetail, String delimiter) {
+        String[] taskDetailArr = taskDetail.split(delimiter);
+        return taskDetailArr;
+    }
+
+    public void addTask(String newTask, TaskType type) {
+        Task typ = getTaskType(newTask, type);
+        taskList[taskCount] = typ;
+        taskCount++;
+    }
+
+    public void listTask() {
+        if (taskCount > 0) {
+            for (int i = 0; i < taskCount; i++) {
+                int j = i + 1;
```

Comment:
I think a more meaningful name (such as taskNumber) would be better instead of j?